src/test/java/nextstep/step2/domain/ParticipantNamesTest.java

```diff
@@ -0,0 +1,19 @@
+package nextstep.step2.domain;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+class ParticipantNamesTest {
+
+    @DisplayName("참여자 수가 0명이면 예외 처리")
+    @Test
+    void participant_count_not_zero() {
+        List<ParticipantName> blankList = new ArrayList<>();
+        Assertions.assertThatThrownBy(() -> ParticipantNames.from(blankList))
+                .isInstanceOf(IllegalArgumentException.class);
```

Comment:
```suggestion
        List<ParticipantName> blankList = new ArrayList<>();

        Assertions.assertThatThrownBy(() -> ParticipantNames.from(blankList))
                .isInstanceOf(IllegalArgumentException.class);
```

이렇게 빈 줄로 검증부를 다른 부분과 분리해 주시면 테스트가 무엇을 검증하는지 훨씬 눈에 잘 들어온다는 점도 참고 부탁드려요.